proto/storage/alert.proto

```diff
@@ -147,7 +147,8 @@ message Alert {
   google.protobuf.Timestamp resolved_at = 17; // @gotags: sensorhash:"ignore"
 
   ViolationState state = 11; // @gotags: search:"Violation State,store" sql:"index=btree"
-  google.protobuf.Timestamp snooze_till = 12;
+  reserved 12;
+  reserved "snooze_till";
```

Comment:
Did not know we could reserve var names too.

Reply:
I didn't either until the proto compatibility checker was yelling at me.